release-notes/release-notes/opensearch-dashboards.release-notes-2.15.0.md

```diff
@@ -0,0 +1,66 @@
+## What's Changed
```

Comment:
Missing PRs
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6613
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6721
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6759
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6811
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6851
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6777

Reply:
Regenerated the doc